src/ExtranetBundle/objects/TCMSTableEditor/TableEditorExtranetUser.php

```diff
@@ -112,6 +112,10 @@ public function LoginAsExtranetUser(): void
         }
 
         $portal = $this->getPortalForExtranetUserId($userId);
+        if (false === $portal) {
```

Comment:
I allowed myself to add this behavioural change since this is my own code to start with: I did not handle the 'not-found' case